i18n-config.js

```diff
@@ -85,6 +85,12 @@ module.exports = {
         short_name: 'TR',
         affiliate_lang: 14,
     },
+    bn: {
+        path: 'bn',
+        display_name: 'বঙ্গালী',
```

Comment:
It will be বাংলা

Reply:
😆  That was fast @habib-deriv . Let me amend that. Thanks